client/docs/statik/init.go

```diff
@@ -1,3 +1,7 @@
 package statik
 
+// unnamed import of statik for swagger UI support
+// override ethermint statik by importing it
+import _ "github.com/tharsis/ethermint/client/docs/statik"
```

Comment:
q: why do we need to run ethermint's before evmos?

Reply:
Because `evmos/cmd/evmosd` indirectly imports `ethermint/app`. Without this import, the execution order of the init methods would be

`evmos/client/docs/statik.init()` --> .... --> `ethermint/client/docs/statik.init()` --> `ethermint/app.init()` -- > `evmos/app.init()` --> `evmos/cmd/evmosd.init()`

`ethermint/client/docs/statik.init()` will overwrite the swagger doc to ethermint's.

Reply:
This import will modify the execution order to

`ethermint/client/docs/statik.init()` --> `evmos/client/docs/statik.init()` --> ....  --> `ethermint/app.init()` -- > `evmos/app.init()` --> `evmos/cmd/evmosd.init()`